packages/teams-js/src/public/authentication.ts

```diff
@@ -341,13 +341,12 @@ export namespace authentication {
     }
   }
 
-  
```

Comment:
This whitespace change was automatically made by VSCode but I'm checking it in otherwise it will keep making it.